src/_playground/App.scss

```diff
@@ -71,16 +71,12 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
         .frDocs-Content {
             flex-grow: 1;
             overflow-y: scroll;
-            padding-bottom: 2rem;
             background-color: white;
-            margin-right: auto;
-            margin-left: 12px;
-            padding-left: 1rem;
-            padding-right: 1rem;
+            padding: 2rem;
 
             .frDocs-Content__header, .frDocs-markdown > h1 {
                 margin: 12px 0;
-                margin-top: 2rem;
+                margin-top: 0;
```

Comment:
Should we just rewrite these lines as `margin: 0 0 12px;`?

Reply:
yup. done.